README.md

```diff
@@ -1,2 +1,137 @@
 # Hello_Microverse2
 Welcome to my Microverse Repo, I hope you will like it.
+
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello_Microverse] <a name="about-project"></a>
+
+> My first project in Microverse
+
+**[Hello_Microverse]** is a basic webpage for Microverse's project
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+
+- **[Responsive]**
+- **[accessible]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+To get a local copy up and running, follow these steps.
+
+```
+ git clone https://github.com/goldfloored/Hello_Microverse2.git
+```
+
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Goldfloored**
+
+- GitHub: [@goldfloored](https://github.com/goldfloored)
+- Twitter: [@AMazgou](https://twitter.com/AMazgou)
+- LinkedIn: [Amine Mazgou](https://www.linkedin.com/in/amine-mazgou-7324b5223/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- **[login_feature]**
+- **[post_sharing]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
```

Comment:
Please add the contributing section header with a message encouraging people to contribute to your project. Please also update the table of contents to include this section